src/js/modules/accounts/components/LoginModalOverlay.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import injectSheet from "react-jss";
+import { Modal } from "react-bootstrap/lib";
+import SignInForm from "./forms/SignInForm";
+
+const styles = {
+  modalStyle: {
+    width: "50%",
+  },
+  modalContent: {
+    fontSize: "30px",
+  },
+};
+
+const LoginModalOverlay = ({
```

Comment:
Change this component to a general modal in core. You can put whatever children you want in it in the component you render it in.